charts/estafette-k8s-hpa-scaler/templates/deployment.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{.Chart.Name | quote}}
+  namespace: {{.Release.Namespace | quote}}
```

Comment:
In helm charts namespace provided during deployment, and normally not templated. I think it's better to remove namespace from template.

https://github.com/helm/helm/issues/5465#issuecomment-473942223

```
In general, templates should not define a namespace. This is because Helm installs objects into the namespace provided with the --namespace flag. By omitting this information, it also provides templates with some flexibility for post-render operations (like helm template | kubectl create -f -)
```

Reply:
thanks!